apps/omar-ui-app/grails-app/assets/javascripts/omar/reachback/reachback.controller.es6

```diff
@@ -0,0 +1,401 @@
+"use strict";
+angular
+    .module("omarApp")
+    .controller("ReachbackController", [
+        "stateService",
+        "$http",
+        "$scope",
+        "wfsService",
+        "mapService",
+        "$stateParams",
+        "$window",
+        "toastr",
+
+        function (
+            stateService,
+            $http,
+            $scope,
+            wfsService,
+            mapService,
+            $stateParams,
+            $window,
+            toastr
+        ) {
+
+            /* jshint validthis: true */
+            let vm = this;
+            vm.userPreferences = AppO2.APP_CONFIG.userPreferences.o2SearchPreference;
+            vm.urlParams = $stateParams;
+
+            // Reachback will be hidden if value is false in application.yml
+            vm.reachbackEnabled = AppO2.APP_CONFIG.params.reachbackEnabled;
+
+            // Reachback URL for retrieving json data from reachback
+            let reachbackUrl = AppO2.APP_CONFIG.params.reachbackUrl;
+
+            let mapVisibility = vm.urlParams.mapVisibility == "true" || vm.userPreferences.mapVisibility;
+            if (!mapVisibility && !vm.urlParams.mapSearch) {
+                setTimeout(function () {
+                    $("a:contains('Filters')").trigger("click");
+                }, 10);
+            }
+
+            let sensorIDTemp = false;
+            let sensorSame = false;
+
+            vm.switchCheckDown = function() {
+                sensorIDTemp = vm.sensorIdCheck;
+                setTimeout(function() {
+                    sensorSame = sensorIDTemp === vm.sensorIdCheck;
+                    sensorIDTemp = vm.sensorIdCheck; }, 10);
+            }
+
+            vm.doUpdateOnNoChange = function() {
+                if (sensorSame && vm.sensorId != '')
+                    vm.updateFilterString();
+                else
+                    vm.sensorIdCheck = sensorIDTemp;
+            }
+
+            // Hide the reachback panel initially
+            $('.JSONPanel').hide();
+
+            // Initialise the cards panel to be selected
+            $( '#button0' ).removeClass();
+            $( '#button0' ).addClass("reachbackTabButtonClicked");
+
+            vm.showCurrentFilter = true;
+            vm.refreshSpin = false;
+            vm.refreshList = function () {
```

Comment:
This can be removed, right?  I don't see it attached to anything in the partial.  The data stays in cache when you flip back to the normal screen/filter?

Reply:
Ya thanks for noticing. Because I just copied the filter.controller code and worked from there, I had to delete a lot of the wfs stuff for the reachback.controller since it doesn't use the wfs service, but didn't catch everything.